CHANGELOG.md

```diff
@@ -29,6 +29,18 @@ instructions](https://nasa.github.io/cumulus/docs/upgrade-notes/upgrade-rds-clus
   - Update `cumulus-rds-tf` module to take additional parameters in support of
     migration from Aurora PostgreSQl v11 to v13.   See Migration Notes for more details.
 
+
+Please note changes in 16.1.4 may not yet be released in future versions, as this
```

Comment:
This should be moved up to near the v16.1.4 header.    Thank you!

Reply:
@Jkovarik correction made, thanks.